console/lockable_trait.rst

```diff
@@ -28,7 +24,7 @@ that adds two convenient methods to lock and release commands::
 
         protected function execute(InputInterface $input, OutputInterface $output)
         {
-            if (!$this->lock()) {
+            if (!$this->acquire()) {
```

Comment:
This doesn't look correct:

https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Console/Command/LockableTrait.php#L33

Reply:
Indeed, my bad... Thank you @apfelbox